pkg/object/azure.go

```diff
@@ -99,7 +103,10 @@ func (b *wasb) Put(key string, data io.Reader) error {
 	if b.sc != "" {
 		options.AccessTier = str2Tier(b.sc)
 	}
-	_, err := b.azblobCli.UploadStream(ctx, b.cName, key, data, &options)
+	resp, err := b.azblobCli.UploadStream(ctx, b.cName, key, data, &options)
+	if resp.RequestID != nil {
```

Comment:
resp could be nil

Reply:
<img width="1897" alt="image" src="https://github.com/juicedata/juicefs/assets/31313340/979b5eaf-a3b0-4289-8425-f878b082640f">
resp is not a pointer